include/aikido/control/ros/Conversions.hpp

```diff
@@ -45,7 +45,7 @@ std::unique_ptr<aikido::trajectory::Spline> toSplineJointTrajectory(
 /// \param[in] trajectory Aikido trajectory to be converted.
 /// \param[in] timestep Timestep between two consecutive waypoints.
 trajectory_msgs::JointTrajectory toRosJointTrajectory(
-    const aikido::trajectory::TrajectoryPtr& trajectory, double timestep);
+    const aikido::trajectory::ConstTrajectoryPtr& trajectory, double timestep);
```

Comment:
Nit: Could we change this to const raw pointer for the same reason above?

Reply:
Same as above